rust-toolchain.toml

```diff
@@ -0,0 +1,2 @@
+[toolchain]
+channel = "nightly"
```

Comment:
Will this affect [users building Yazi](https://yazi-rs.github.io/docs/installation#build-from-source)? Generally users should use the latest stable version of Rust to build it, not the nightly version.

Reply:
sorry I didn't see this! I just found the rustfmt's nightly features doesn't work on stable channel, so I subconsciously think we are using nightly version of Rust. But now when I run format from editor, it won't add the +nightly option, so the format will be broken. I should find if there is any better way to fix this from my editor config or something else.

Reply:
No problem! In that case I'm going to close this PR, @zooeywm thanks for the effort!